i18n/zh/docusaurus-plugin-content-docs/current/architecture/cloud/edge_controller.md

```diff
@@ -3,57 +3,56 @@ title: Edge Controller
-   - **Initialized**: It means that all Init containers in the pod have started successfully
-   - **ContainersReady**: It indicates whether all containers in the pod are ready
-- **Below is the information for PodStatus**
-   - **PodPhase**: Current condition of the pod
-   - **Conditions**: Details indicating why the pod is in this condition
-   - **HostIP**: IP address of the host to which pod is assigned
-   - **PodIp**: IP address allocated to the Pod
-   - **QosClass**: Assigned to the pod based on resource requirement
+
+## Upstream Controller：
+### 同步监视并更新资源和事件的状态
+
+- Upstream Controller接收来自EdgeCore的消息，并将更新同步到K8S Api-server
+- 创建停止通道以分发和停止处理Pod,ConfigMap节点和Secret事件
+- 创建消息通道以更新NodeStatus,PodStatus,Secret和ConfigMap相关事件
+- 获取Pod状态信息，如Ready,Initialized,PodScheduled和Unschedulable详细信息
+- **以下是 PodCondition 的信息**
+   - **Ready**：PodReady表示Pod能够提供服务请求，并应添加到所有匹配服务的负载均衡池中
+   - **PodScheduled**：它表示此Pod的调度过程状态
+   - **Unschedulable**：表示调度器当前无法调度此Pod，可能是由于集群中的资源不足
+   - **Initialized**：表示Pod中的所有初始化容器均已成功启动
+   - **ContainersReady**：表示Pod中的所有容器是否都已准备好
+- **以下是 PodStatus 的信息**
+   - **PodPhase**：Pod的当前状态
+   - **Conditions**：指示Pod处于此状态的详细信息
+   - **HostIP**：分配给Pod的主机的IP地址
+   - **PodIp**：分配给Pod的IP地址
+   - **QosClass**：根据资源需求分配给Pod的QoS类别
 
    ![Upstream Controller](/img/edgecontroller/UpstreamController.png)
 
-## Controller Manager:
-### Creates manager interface and implements ConfigmapManager, LocationCache and podManager
+## 控制器管理器(Controller Manager)：
+### 创建管理接口并实现 ConfigmapManager、LocationCache 和 PodManager
 
-- Manager defines the interface of a manager, ConfigManager, Podmanager, secretmanager implements it
-- Manages OnAdd, OnUpdate and OnDelete events which will be updated to the respective edge node from the K8s Api-server
-- Creates an eventManager(configMaps, pod, secrets) which will start a CommonResourceEventHandler, NewListWatch and a newShared Informer for each event to sync(add/update/delete)event(pod, configmap, secret) to edgecore via cloudHub
-- **Below is the List of handlers created by the controller Manager**
-   - **CommonResourceEventHandler**: NewcommonResourceEventHandler creates CommonResourceEventHandler which is used for Configmap and pod Manager
-   - **NewListWatch**: Creates a new ListWatch from the specified client resource namespace and field selector
-   - **NewSharedInformer**: Creates a new Instance for the Listwatcher
\ No newline at end of file
+- Controller Manager定义了manager和ConfigManager,PodManager,SecretManager的接口,并实现了该接口
+- 管理OnAdd,OnUpdate和OnDelete事件，并将这些事件将从K8s Api-server更新到相应的边缘节点
+- 创建一个EventManager(ConfigMap,Pod,Secret)，它将为每个事件启动一个CommonResourceEventHandler、新的ListWatch和一个newShared Informer，以通过cloudHub将事件添加/更新/删除）同步到EdgeCore
+- **以下是Controller Manager创建的处理程序列表**
+   - **CommonResourceEventHandler**：NewcommonResourceEventHandler创建了用于ConfigMap和Pod管理的CommonResourceEventHandler
+   - **NewListWatch**：从指定的客户端资源命名空间和字段选择器创建一个新的ListWatch
+   - **NewSharedInformer**：为ListWatcher创建一个新的实例
```

Comment:
![image](https://github.com/user-attachments/assets/387fa242-24cb-4795-8b5b-2b70365e406d)
ditto

Reply:
![image](https://github.com/user-attachments/assets/4661bb40-5868-47e2-96f4-44023f87585e)
Update to adjust the format